test/system/trader.js

```diff
@@ -185,15 +185,17 @@ class Trader {
     }, options);
   }
 
-  placeMarketOrder(side, quantity, options) {
+  placeMarketOrder(side, quantity, market, options) {
```

Comment:
Won't this change break all the other uses of placeMarketOrder (e.g. two-trades)?

Reply:
No it will not break .. if we don't specify market(from other places like from two-trades) it will get it's  market value from createOrder function

Reply:
Fair enough